proposal/package-importer.d.ts.md

```diff
@@ -216,17 +216,27 @@ using symlinks if this behavior is desired.
 
 [ECMAScript modules]: https://nodejs.org/api/esm.html#no-node_path
 
+The [Node resolution algorithm] requires a `parentURL`, used for determining
+where in the file system to start searching for a module. When compiling a
```

Comment:
"...to start searching for a module **if a `pkg:` URL is being resolved somewhere other than a file on disk**."

Reply:
In https://github.com/sass/sass/pull/3748/commits/3afb76cc2bf5df211040bfb51323f4e5a0fd074e, I changed this, with the addition of "...is
being resolved **in a source** somewhere other than..." to clarify we're talking about resolved **from** as opposed to the resolution destination.